tests/cli/conftest.py

```diff
@@ -27,6 +27,19 @@ def run_in_default_project(testdir: Testdir) -> Callable[..., RunResult]:
     os.environ["LOG_LEVEL"] = "ERROR"
     testdir.run("rasa", "init", "--no-prompt")
 
+    def do_run(*args):
+        args = ["rasa"] + list(args)
+        result = testdir.run(*args)
+        os.environ["LOG_LEVEL"] = "INFO"
+        return result
+
+    return do_run
+
+
+@pytest.fixture
+def run_in_default_project_with_info(testdir: Testdir) -> Callable[..., RunResult]:
```

Comment:
why do we need this? Shouldn't the default be `info` anyway?

Reply:
No. The first thing that `run_in_default_project` does is to set `os.environ["LOG_LEVEL"] = "ERROR"`. This is used in dozens of tests, but it didn't work for mine. So I created this new function `run_in_default_project_with_info`. 
We could rename `run_in_default_project` into `run_in_default_project_with_error_output_only` and then rename `run_in_default_project_with_info` to `run_in_default_project`.
Alternatively, we could rename `run_in_default_project_with_info` into `run_in_default_project_verbose`. What do you think?

Reply:
I think we should not use this at all since it trains a model and this makes our CI slow. You can do it the same way as here https://github.com/RasaHQ/rasa/pull/5243/files#diff-86687b69c02e3fbdf8fcc98e3b4ab350R163 + also use the caplog fixture for the log level https://docs.pytest.org/en/latest/logging.html#caplog-fixture